### ISSUE TITLE
Plugin not loading since upgrade to Obsidian v1.1.15

### ISSUE THREAD

**Issue by kristanner2015** [vrtmrz/obsidian-livesync#183]
Running Obsidian AppImage v1.1.5.  Ever since this automatic upgrade my livesync plugin is not loading properly.  It is listed in the list of community plugins in Settings, but there are no icons for initiating the sync or viewing the logs in the left-hand toolbar.  Also there is no indication in the status bar that a sync happens even when I update local files.

I am syncing to a local couchDB instance.  Syncing was working very well before this upgrade, but not at all now.

When I launch Obsidian from the command line the following is displayed on stdout:
```
❯ bin/Obsidian.AppImage
2023-02-27 20:23:49 Loading updated app package /home/kris/.config/obsidian/obsidian-1.1.15.asar
2023-02-27 20:23:50 Checking for update using Github
2023-02-27 20:23:51 Success.
2023-02-27 20:23:51 Latest version is 1.1.15
2023-02-27 20:23:51 App is up to date.
[19903:0227/122356.065237:ERROR:leveldb_factory.cc(88)] Failed to open LevelDB database from /home/kris/.config/obsidian/IndexedDB/app_obsidian.md_0.indexeddb.leveldb,IO error: /home/kris/.config/obsidian/IndexedDB/app_obsidian.md_0.indexeddb.leveldb/LOCK: File currently in use. (ChromeMethodBFE: 15::LockFile::2)
```
The last line repeats 20-30 times.

**Comment by kristanner2015** [comment]
Version of livesync plugin installed is 0.17.27

**Comment by vrtmrz** [comment]
Thank you for pointing it out to me! I did not reproduce it yet, but I think it is a nightmare for Linux.

Actually, even in windows, the error database had been shown at booting. I supposed that I had ignored that and it made this.

I will try to fix it soon, but it will be late to check on Linux. However, anyway, we cannot use the plugin on Linux now, I will release the fixed version. Sorry for writing like a draft!

**Comment by vrtmrz** [comment]
@kristanner2015 
Version 0.17.28 has been released. This version suppresses the behaviour of repeatedly closing and reopening the database. (And the logic for compatibility with the older databases has been removed too.)
I hope that it would work properly in a Linux environment.

I would like to apologise for your inconvenience, but could you please try on Linux? It will be late at night and my Linux device has an Arm chip, I am not sure that it has been even reproduced.

**Comment by kristanner2015** [comment]
I have upgraded to 0.17.28 and restarted the Obsidian app completely.  I still don't see the buttons to initiate a sync or the sync status on the status line.  In the Settings dialog -> Community Plugins, I see "Self-Hosted LiveSync" is listed and the toggle button next to it is "ON".  However, there is no "Options" button next to it.  Also, "Self-Hosted LiveSync" does not show up in the list of Community Plugins on the left-hand side of the Settings dialog, meaning I can't click on it there to change options.
So, perhaps the plugin is loading, but it is not functional - I can't change any options or trigger a sync.  There is no indication that automatic/scheduled syncs are happening either.
Hope all that makes sense to you.

**Comment by vrtmrz** [comment]
Thank you for checking it around again! However, I regret to say that it looks to run correctly on my devices (WSL2 of x86, and Chromebook of Arm). We still did not catch the reproducibility. 

This is not the way to solve straightforward this, but could you please try changing the name of the vault? If it works, actually something corrupted. 

In addition to that, May I ask you to see what is shown (Or, what process has not been finished) on the DevTools? It can be seen by pressing `Ctrl+Shift+I` after Obsidian has been launched. Setting `showVerboseLog` to `true` on `data.json` is recommended.

**Comment by kristanner2015** [comment]
I've resolved this issue.  Turns out that error message I posted first was the key.  I noticed that the .config/obsidian/IndexedDB/app_obsidian.md_0.indexeddb.leveldb/LOCK file had a timestamp of 2023/01/03.  Since it was a 0-byte file I simply removed it.  Starting Obsidian after that showed that the LiveSync plugin is enabled again.  Unfortunately, in an attempt to fix the problem today I resorted to uninstalling the plugin, so now I've got to set up the database connection again.  But at least I can now get to the plugin settings to make those changes.
Thanks for taking the time to investigate.
